cli/units_cmd.go

```diff
@@ -111,6 +119,21 @@ func scanUnitsIntoConfig(cfg *config.Repository, quiet bool) error {
 	return nil
 }
 
+func checkRegularExecutableFile(name string) error {
+	switch info, err := os.Stat(name); {
+	case err != nil:
+		return err
+	case info.IsDir():
+		return fmt.Errorf("a directory")
+	case !info.Mode().IsRegular():
+		return fmt.Errorf("not a regular file")
+	case info.Mode().Perm()&0x1 == 0:
```

Comment:
This is just a minor style suggestions, but unix permissions are usually expressed using octal numbers (e.g., `0777`). So, this could be rewritten to use `01` or `0001`. Do you think that's better than hex or not?

Also, do you want to check just the last bit for "others", what about "owner" and "group" executable bits?